src/handwritten/task.py

```diff
@@ -76,6 +77,11 @@ def __init__(self, new_task_name='', *, grpc_options=None):
         self._saved_name = new_task_name  # _initialize sets this to the name assigned by DAQmx.
         self._grpc_options = grpc_options
 
+        self._done_event_handler = None
+        self._every_n_acquired_event_handler = None
+        self._every_n_transferred_event_handler = None
+        self._signal_event_handler = None
```

Comment:
Using 4 separate member variables makes this more complicated than it has to be. We should use a dict. I'll address this in a follow-up PR.

Reply:
#401